definitions/grib2/template.4.statistical.def

```diff
@@ -56,8 +56,10 @@ if (numberOfTimeRange == 1 || numberOfTimeRange == 2) {
     "instant" = {typeOfStatisticalProcessing=255;}
     "avg"     = {typeOfStatisticalProcessing=0;typeOfTimeIncrement=2;}
     "avg"     = {typeOfStatisticalProcessing=0;typeOfTimeIncrement=3;}
+    "avg"     = {typeOfStatisticalProcessing=0;typeOfTimeIncrement=255;}
     "avgd"    = {typeOfStatisticalProcessing=0;typeOfTimeIncrement=1;}
     "accum"   = {typeOfStatisticalProcessing=1;typeOfTimeIncrement=2;}
+    "accum"   = {typeOfStatisticalProcessing=0;typeOfTimeIncrement=255;}
```

Comment:
Why is the typeOfStatisticalProcessing=0 ( which means average ) for the entry "accum"?

Reply:
Wow, I actually managed to make a mistake in a PR of 2 lines of code. I do apologize, and thank you for catching that! I have amended the commit to fix it.